canvas_modules/common-canvas/__tests__/_utils_/common-canvas-utils.js

```diff
@@ -110,6 +110,88 @@ export function createIntlCommonCanvas(
 	return wrapper;
 }
 
+export function createIntlCommonCanvasRTL(
+	config,
+	contextMenuHandler,
+	beforeEditActionHandler,
+	editActionHandler,
+	clickActionHandler,
+	decorationActionHandler,
+	selectionChangeHandler,
+	tipHandler,
+	showBottomPanel,
+	showRightFlyout,
+	toolbarConfig,
+	notificationConfig,
+	contextMenuConfig,
+	canvasController) {
+
+
+	const wrapper = render(
+		<IntlProvider key="IntlProvider1" locale={ locale } messages={messages}>
+			<CommonCanvas
+				config={config}
+				contextMenuHandler={contextMenuHandler}
+				beforeEditActionHandler={beforeEditActionHandler}
+				editActionHandler={editActionHandler}
+				clickActionHandler={clickActionHandler}
+				decorationActionHandler={decorationActionHandler}
+				selectionChangeHandler={selectionChangeHandler}
+				tipHandler={tipHandler}
+				toolbarConfig={toolbarConfig}
+				notificationConfig={notificationConfig}
+				contextMenuConfig={contextMenuConfig}
+				showRightFlyout={showRightFlyout}
+				showBottomPanel={showBottomPanel}
+				canvasController={canvasController}
+			/>
+		</IntlProvider>
+	);
+	return wrapper;
+}
+
+export function createIntlCommonCanvasRTLRerender(
```

Comment:
What is the difference between `createIntlCommonCanvasRTL` and `createIntlCommonCanvasRTLRerender` methods?

Reply:
Difference between` createIntlCommonCanvasRTL` and `createIntlCommonCanvasRTLRerender` is that we are passing rerender as argument for `createIntlCommonCanvasRTLRerender`